jsk_arc2017_common/node_scripts/set_location_in_rosparam.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
```

Comment:
what I want to do is set param from json.
I don't want to modify last year a lot to ensure it works, and I want to set bin_contents in`/bin_contents` ns.
As a result, i need a script like this.